languagescool/source/scss/contact_us.scss

```diff
@@ -0,0 +1,141 @@
+.mainform {
+    &__container {
+        max-width: 1024px;
+        margin: 0 auto;
+    }
+
+    &__info {
+        text-align: center;
+        margin: 50px 0px;
+    }
+
+    & h1 {
+        margin-bottom: 20px;
+        font-weight: bold;
+    }
+
+    &__form {
+        max-width: 900px;
+        margin: 0 auto;
+    }
+}
+.info {
+    &__phone {
+        margin-bottom: 20px;
+        & p {
+            font-style: italic;
+            font-weight: bold;
+        }
+    }
+
+    &__address {
+        & p {
+            font-style: italic;
+            font-weight: bold;
+        }
+    }
+}
+.map {
+    max-width: 900px;
+    padding: 0px 20px;
+    margin: 0 auto;
+    & h2 {
+        text-align: center;
+        font-size: 22px;
+        font-weight: bold;
+        margin-bottom: 20px;
+    }
+    &__frame {
+        width: 100%;
+        height: 400px;
+        border: 1px solid #000;
+        & iframe {
+            width: 100%;
+            height: 100%;
+        }
+    }
+}
+.form {
+    display: block;
+    width: 400px;
+    height: 650px;
+    position: relative;
+    animation: largetonormal 3s ease-in 1;
+
+    @keyframes largetonormal {
+        0% {
+            transform: scale(2);
+            opacity: 0;
+        }
+        100% {
+            transform: scale(1);
+            opacity: 1;
+        }
+    }
+
+    &__form {
+        display: block;
+        margin-bottom: 50px;
+        padding: 0px 20px;
+        width: 400px;
+        font-weight: bold;
+        font-size: 18px;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate3d(-50%, -50%, 0);
+        & label {
```

Comment:
please, fix it here and everywhere in other places

`&` is not needed here

Reply:
Ok